src/lib.rs

```diff
@@ -77,23 +73,64 @@ pub enum WildcardError {
         /// First position of the error in the wildcard.
         position: usize,
         /// Description of the error.
-        message: String,
+        message: &'static str,
     },
     /// The wildcard matching not configured correctly, as it contains repeated special symbols
     /// (either the metasymbols or the escape symbol).
     #[error("invalid configuration of special symbols")]
     InvalidSpecialSymbolsConfiguration,
 }
 
+/// This trait defined the alphabet a wildcard can be used on.
+pub trait WildcardSymbol: Eq + Copy {
+    /// Default metasymbol that matches on sequences of arbitrary length.
+    ///
+    /// Normally this is `*`.
```

Comment:
s/Normally/By default?

Reply:
I'm not talking about the default, but the typical character in "wildcard language". The default depends on the alphabet. In fact, let me change normally to typically because it captures better what I'm trying to say.